shell/platform/android/io/flutter/plugin/platform/PlatformViewFactory.java

```diff
@@ -29,6 +31,16 @@ public PlatformViewFactory(MessageCodec<Object> createArgsCodec) {
      */
     public abstract PlatformView create(Context context, int viewId, Object args);
 
+    /**
+     * Used to customize the Theme of Presentation. This is useful when need a translucent AndroidView.
+     * Return 0 by default, will use the theme defined by com.android.internal.R.attr.presentationTheme attribute.
+     * @return A style resource describing the theme to use for the window.
+     */
+    @StyleRes
+    protected int getPresentationTheme() {
```

Comment:
I don't really know how this works. 

Though it seems a bit unfortunate to create this parallel mechanism to side-load this. Can it not be passed via args from the Flutter side and then use context.setTheme or something instead?